partiql-lang/src/test/kotlin/org/partiql/lang/errors/ParserErrorsTest.kt

```diff
@@ -12,7 +12,7 @@
  *  language governing permissions and limitations under the License.
  */
 
-package org.partiql.lang.errors
+package org.partiql.errors
```

Comment:
For all the `partiql-lang/src/test/kotlin/org/partiql/lang/errors` tests, was this change intentional?

Reply:
No it was the find-replace. It shouldn't matter where the tests live, but I think we should keep them in the existing path to minimize the diff